src/backend/cdb/dispatcher/cdbdisp_query.c

```diff
@@ -906,13 +909,13 @@ buildGpQueryString(DispatchCommandQueryParms *pQueryParms,
 		sizeof(currentUserId) +
 		sizeof(n32) * 2 /* currentStatementStartTimestamp */ +
 		sizeof(command_len) +
+		sizeof(withPlanAndDdescValue) +
```

Comment:
can `withPlanAndDdescValue ` be 16-bit positive integers ?

Reply:
I further change this to a `char`.